### PR TITLE
docs: footer cleanup

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

Switch maintainer information

Closes: #XXX

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ module.exports = {
         }
       ],
       smallprint:
-          'The development of the Tendermint project is led primarily by Tendermint Inc., the for-profit entity which also maintains this website. Funding for this development comes primarily from the Interchain Foundation, a Swiss non-profit.',
+          'The development of the Tendermint project is led primarily by [Interhcain GmbH](https://interchain.berlin/). Funding for this development comes primarily from the Interchain Foundation, a Swiss non-profit.',
```
Comment (alessio) [inline docs/.vuepress/config.js:108]:
```suggestion
          'The development of the Tendermint project is led primarily by [Interhcain GmbH](https://interchain.berlin/). Funding for this development comes primarily from the Interchain Foundation, a Swiss non-profit. The Tendermint trademark is owned by Tendermint Inc.',
```

I'd like to hear @nylira's thoughts on this :+1:

Comment (tessr) [reply docs/.vuepress/config.js:108]: I like this. @nylira, what do you think?

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -105,7 +105,7 @@ module.exports = {
         }
       ],
       smallprint:
-          'The development of the Tendermint project is led primarily by Tendermint Inc., the for-profit entity which also maintains this website. Funding for this development comes primarily from the Interchain Foundation, a Swiss non-profit.',
+          'The development of the Tendermint project is led primarily by [Interchain GmbH](https://interchain.berlin/). Funding for this development comes primarily from the Interchain Foundation, a Swiss non-profit.',
```
Comment (alessio) [inline docs/.vuepress/config.js:108]:
```suggestion
          'The development of the Tendermint project is led primarily by [Interchain GmbH](https://interchain.berlin/). Funding for this development comes primarily from the Interchain Foundation, a Swiss non-profit. The Tendermint trademark is owned by Tendermint Inc., the for-profit entity that also maintains this website.',
```

Comment (alessio) [reply docs/.vuepress/config.js:108]: @tessr @nylira does this work for you?

Comment (tessr) [reply docs/.vuepress/config.js:108]: I like this!

Comment (marbar3778) [reply docs/.vuepress/config.js:108]: Committing this suggestion. @nylira does this work for you?

**Review by nylira — APPROVED** [review]

**Pull request merged**